modules/search/pages/search-index-params.adoc

```diff
@@ -0,0 +1,1004 @@
+
+For more information, see <<types,>>.
+
+|====
+
+[#analyzers]
+== Analyzers Object 
+
+The `analyzers` object contains any custom analyzers defined for a Search index.
+
+[source,json]
+----
+include::example$simple-search-index-payload.jsonc[lines=23..37]
+----
+
+TIP: To view the entire JSON payload, click btn:[View].
+
+The `analyzers` object is a child object of the <<analysis,analysis object>>. It contains any number of <<analyzer-name,{analyzer_name} objects>>:
+
+[cols="1,1,1,2"]
+|====
+|Property |Type |Required? |Description
+
+|\{analyzer_name\} |Object |Yes a|
+
+Set the name of this object to the name you want for your custom analyzer. 
+
+You can reference the `\{analyzer_name\}` object elsewhere in your Search index definition to use the analyzer. 
+
+For more information about the properties in an `\{analyzer_name\}` object, see <<analyzer-name,>>.
+|====
+
+[#analyzer-name]
+=== \{Analyzer_name\} Object
+
+The `\{analyzer_name\}` object defines a custom analyzer for a Search index: 
+
+[source,json]
+----
+include::example$simple-search-index-payload.jsonc[lines=24..36]
+----
+
+An `\{analyzer_name\}` object is a child object of the <<analyzers,>>. It contains the following properties:
+
+[cols="1,1,1,2"]
+|====
+|Property |Type |Required? |Description
+
+|token_filters |Array |Yes a|
+
+An array of strings that contains the token filters for the custom analyzer. 
+
+For more information about the token filters you can define in a Search index JSON payload, see <<token_filters,>>.
+
+You can also use one of the xref:default-token-filters-reference.adoc[default token filters].
+
+|char_filters |Array |Yes a|
+
+An array of strings that contains the character filters for the custom analyzer. 
+
+For more information about the character filters you can define in a Search index JSON payload, see <<char_filters,>>.
+
+You can also use one of the xref:default-character-filters-reference.adoc[default character filters] available. 
+
+|type |String |Yes |The `type` is always `"custom"`.
+
+|tokenizer |String |Yes |The selected tokenizer for the custom analyzer. 
+
+|====
+
+[#char_filters]
+== Char_filters Object 
+
+The `char_filters` object contains any custom character filters defined for a Search index:
+
+[source,json]
+----
+include::example$simple-search-index-payload.jsonc[lines=38..44]
+----
+
+TIP: To view the entire JSON payload, click btn:[View].
+
+The `char_filters` object is a child object of the <<analysis,analysis object>>. It contains any number of <<char-name,{char_filter_name} objects>>: 
+
+[cols="1,1,1,2"]
+|====
+|Property |Type |Required? |Description
+
+|\{char_filter_name\} |Object |Yes a|
+
+Set the name of this object to the name you want for your custom character filter. 
+
+You can reference the `\{char_filter_name\}` object elsewhere in your Search index definition to use the character filter. 
```

Comment:
do you need the escapes inside backticks?

Reply:
It wasn't displaying properly otherwise.